gui/convoi_stops_list_t.cc

```diff
@@ -0,0 +1,158 @@
+/*
+ * This file is part of the Simutrans project under the Artistic License.
+ * (see LICENSE.txt)
+ */
+
+#include "../simline.h"
+#include "../simcolor.h"
+#include "../simhalt.h"
+#include "../simworld.h"
+#include "../simmenu.h"
+#include "../simconvoi.h"
+#include "../display/simgraph.h"
+#include "../display/viewport.h"
+
+#include "../utils/simstring.h"
+#include "../utils/cbuffer_t.h"
+
+#include "../dataobj/schedule.h"
+#include "../dataobj/translator.h"
+#include "../dataobj/loadsave.h"
+
+#include "../player/simplay.h"
+
+#include "../tpl/vector_tpl.h"
+
+#include "convoi_stops_list_t.h"
+#include "halt_info.h"
+
+#include "components/gui_image.h"
+#include "components/gui_textarea.h"
+#include "simwin.h"
+
+static karte_ptr_t welt;
+class gui_halt_detail_t;
+
+/**
+ * One entry in the list of schedule entries.
+ */
+class gui_convoi_stops_list_t : public gui_aligned_container_t, public gui_action_creator_t
```

Comment:
"list_t" だと各エントリが並んでいるリスト全体のコンポーネントのように見えます。このコンポーネントの実態はitemです。また、prefix `gui` は共通コンポーネントにつけるprefixです。
```suggestion
class convoi_stops_list_item_t : public gui_aligned_container_t, public gui_action_creator_t
```